tests/test_grading.py

```diff
@@ -180,3 +183,75 @@ def test_scores_with_generate(self, generate=False):
             backup_creators.extend(s.backup.owners())
 
         self.assertEquals(len(backup_creators), len(csv_rows) - 1)
+
+    def test_publish_grades(self):
+        points = [20.0, 2.0, 15.0, 1.0, 17.0, 0.0, 20.0, 1.0]
+        scores, i, users = {}, 0, [self.user1, self.user3]
+        for score_kind in ['Total', 'Composition']:
+            for user in users:
+                for assign in self.active_assignments:
+                    backup = assign.final_submission(assign.active_user_ids(user.id))
+                    scores = Score(backup_id=backup.id, kind=score_kind, score=points[i],
```

Comment:
It doesn't seem like `i` is used at all (Only set for 0). Should it be? 

Can everything just have the same score?